packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -483,16 +484,24 @@ export function TransferPanelMain({
           String(destinationAddress),
           isDepositMode ? l2.provider : l1.provider
         )
-        if (
-          // Destination address is not required for EOA wallets
-          (!isSmartContractWallet && !destinationAddress) ||
+        if (isSmartContractWallet) {
```

Comment:
When the wallet is SC wallet and connected to the bridge, the UI is initialized with an error below the destination field by default. Ideally, this error should come only when a user forgets to enter the address in this case. Wdyt?
<img width="554" alt="image" src="https://user-images.githubusercontent.com/7558499/232096057-336d5bae-9e2e-4ee8-9893-5232fe29ce8a.png">

Reply:
Good point! Changed it to `required` error message if the input is empty & connected to contract wallet
<img width="1078" alt="Screenshot 2023-04-19 at 13 45 18" src="https://user-images.githubusercontent.com/47932951/233064996-d008c64e-c2f7-4298-ae8f-4bba45611edb.png">